gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html

```diff
@@ -402,9 +402,6 @@ <h3 class="portal__form__card__title">Documentation</h3>
         <h2>New Developer Portal</h2>
         <mat-card class="portal__form__card">
           <mat-card-content formGroupName="portalNext">
-            <gio-banner-info>
-              {{ integrationsService.bannerMessages.techPreview }}
-            </gio-banner-info>
```

Comment:
Please, do no remove tech preview for portal stuff 🙏🏻

Reply:
its a good point, however this is "tightly coupeled" with integrations. I am removing tech preview for integrations, so this should be removed form integrations service.
 If we need "tech preview" for any different functionality(-ties) we need to move it out from integration service to more generic place.